packages/metal-dom/src/keyboardEvents.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+import { object } from 'metal';
+
+export const KEYMAP = {
+  'BACKSPACE': 8,
```

Comment:
Yeah, you both are right, guys. I did not remember that we can achieve this behavior using `import`. Thx.